src/options.js

```diff
@@ -61,6 +61,7 @@ export const SPEED = 1
 export const TOTAL_TICKS = 1600
 export const TICKS_TO_RECOVER = 500
 export const TICKS_TO_INCUBATE = 100
+export const DECELERATION_WHEN_SICK = 100
```

Comment:
Change the name of the option `DECELERATION_WHEN_SICK` as it's not used when sick, only when recovered.

Reply:
Yep

Reply:
Change moved forward to next PR.